packages/studio/src/components/common/StreamScopeField.tsx

```diff
@@ -39,13 +39,14 @@ export default function StreamScopeField({
     },
     [options]
   );
+  const displayedSelectedItems = options.length ? selectedIds : [];
 
   return (
     <>
       <StreamScopeFieldLabel streamScopeField={streamScopeField} />
```

Comment:
I think this is a similar situation as [above](https://github.com/yext/studio/pull/374#discussion_r1328736866)-- I could hoist the logic up to `StreamScopePicker` but the logic would still be separate between `EntityIdField` and `StreamScopeField` since their data are structured differently.

Reply:
Oh, I meant could we put the `StreamScopeFieldLabel` Component under `src/components/common`. Apologies for not being clearer!

Reply:
Gotcha! its moved